cli: unable to run profile with attached services

```
⏚ [adam:~/Projects/e2e-testing/cli] gh-fix-933 11s 130 ± op run profile fleet -s elastic-agent
fleet_elasticsearch_1 is up-to-date
fleet_package-registry_1 is up-to-date
fleet_kibana_1 is up-to-date
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/elastic/e2e-testing/cli/cmd.buildRunProfileCommand.func1(0xc00078a500, 0xc00069df20, 0x0, 0x2)
        /Users/adam/Projects/e2e-testing/cli/cmd/run.go:103 +0x7b5
github.com/spf13/cobra.(*Command).execute(0xc00078a500, 0xc00069dee0, 0x2, 0x2, 0xc00078a500, 0xc00069dee0)
        /Users/adam/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0x4ea8120, 0x8, 0x18, 0xc00069de60)
        /Users/adam/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/adam/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/elastic/e2e-testing/cli/cmd.Execute()
        /Users/adam/Projects/e2e-testing/cli/cmd/root.go:24 +0x42
main.main()
        /Users/adam/Projects/e2e-testing/cli/main.go:17 +0x20
```

--- a/cli/cmd/run.go
+++ b/cli/cmd/run.go
@@ -6,6 +6,7 @@ package cmd
 
 import (
 	"context"
+	"os"
 	"strings"
 
 	"github.com/elastic/e2e-testing/cli/config"
@@ -78,7 +79,11 @@ func buildRunProfileCommand(key string, profile config.Profile) *cobra.Command {
 	return &cobra.Command{
 		Use:   key,
 		Short: `Runs the ` + profile.Name + ` profile`,
-		Long:  `Runs the ` + profile.Name + ` profile, spinning up the Services that compound it`,
+		Long: `Runs the ` + profile.Name + ` profile, spinning up the Services that compound it
+
+Example:
+  go run main.go run profile fleet -s elastic-agent:8.0.0-SNAPSHOT
+`,
 		Run: func(cmd *cobra.Command, args []string) {
 			serviceManager := services.NewServiceManager()
 
@@ -99,6 +104,13 @@ func buildRunProfileCommand(key string, profile config.Profile) *cobra.Command {
 
 				for _, srv := range services {
 					arr := strings.Split(srv, ":")
+					if len(arr) != 2 {
+						log.WithFields(log.Fields{
+							"profile":  key,
+							"services": servicesToRun,
+						}).Error("Unable to determine the <image>:<tag>, please make sure to use a known docker tag format, eg. `elastic-agent:8.0.0-SNAPSHOT`")
+						os.Exit(1)
+					}
 					image := arr[0]
 					tag := arr[1]
 

